src/package_manager.rs

```diff
@@ -189,15 +186,15 @@ impl PackageManager {
     /// Update one or all package using the package manager.
     pub fn update(&self, instruction: &Update) -> DepotResult<()> {
         let result = match self {
-            PackageManager::Pacman => run_command!("pacman", "-Syu", "-S", &instruction.package),
-            PackageManager::Yay => run_command!("yay", "-Syu", "-S", &instruction.package),
-            PackageManager::Apk => run_command!("apk", "upgrade", "upgrade", &instruction.package),
+            PackageManager::Pacman => run_command!(self, "-Syu", "-S", &instruction.package),
+            PackageManager::Yay => run_command!(self, "-Syu", "-S", &instruction.package),
+            PackageManager::Apk => run_command!(self, "upgrade", "upgrade", &instruction.package),
             PackageManager::AptGet => {
-                run_command!("apt-get", "upgrade", "upgrade", &instruction.package)
+                run_command!(self, "upgrade", "upgrade", &instruction.package)
             }
```

Comment:
Why not in a single line like every other package manager ? 
Something like this :
```rust
PackageManager::AptGet => run_command!(self, "upgrade", "upgrade", &instruction.package)
```

Reply:
blame the rust formatter for that, the line was too long so it decided to put it on multiple lines

Reply:
Oh, I see